R/data-process.R

```diff
@@ -391,25 +393,31 @@ gen_data_weighted_burden_spline <- function(data_weighted_burden_wide_collapse)
   ## Fit spline and predict
   spline_fit_mean <- splines::interpSpline(
     as.numeric(unlist(data_weighted_burden_mean[config$group])),
-    as.numeric(unlist(data_weighted_burden_mean[burden_spline[[1]]$burden_mean])))
+    as.numeric(unlist(data_weighted_burden_mean[burden_spline[[1]]$burden_mean]))
+  )
   data_weighted_burden_spline[burden_spline[[1]]$burden_mean] <- stats::predict(
     spline_fit_mean,
-    as.numeric(unlist(data_weighted_burden_spline[config$group])))$y
+    as.numeric(unlist(data_weighted_burden_spline[config$group]))
+  )$y
 
   spline_fit_min <- splines::interpSpline(
     as.numeric(unlist(data_weighted_burden_min[config$group])),
-    as.numeric(unlist(data_weighted_burden_min[burden_spline[[2]]$burden_min])))
+    as.numeric(unlist(data_weighted_burden_min[burden_spline[[2]]$burden_min]))
```

Comment:
Did I suggest changing this in the other PR? For example, change data_weighted_burden_min to data_weighted_burden_ci_low

Reply:
If not, should it be the below?
```suggestion
    as.numeric(unlist(data_weighted_burden_ci_low[burden_spline[[2]]$burden_ci_low]))
```